modules/transforms_update_meta_data.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<ol><li>"child class" -&gt; "sub class", or "inherit from <code>torch.Tensor</code>".</li><li>Could you please help add the "open in colab" button? Refer to: https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb</li></ol><p> </p><p>Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>